packages/kit/test/apps/basics/test/test.js

```diff
@@ -1909,10 +1903,7 @@ test.describe('Actions', () => {
 		expect(await page.textContent('pre.formdata2')).toBe(JSON.stringify(null));
 
 		await page.type('input[name="username"]', 'foo');
```

Comment:
@benmccann Is there a specific reason we're using [`type`](https://playwright.dev/docs/api/class-locator#locator-type) not [`fill`](https://playwright.dev/docs/api/class-locator#locator-fill)?

`fill` seems more robust, but I wasn't sure if we were deliberately not using it precisely _because_ of the extra checks it carries out.

Reply:
It's also used in some of other flaky tests here so I'm thinking it could be a culprit.

Reply:
I didn't write this test, so I'm not sure. I don't know why `fill` would be more robust, but at the same time this test is rather flaky so maybe it doesn't hurt to try it